### DISCUSSION TITLE
Tailwind with Codekit or PrePros

### DISCUSSION THREAD

**Discussion by xcrap** [tailwindlabs/tailwindcss#1473] [General]
Hi, instead of using the Webpack or Gulp or any of these popular bundlers or task runners I'm using Codekit for years. Is there any good way to integrate with this software? Codekit always compiles and optimizes my SCSS to a final CSS.

So I'm using CodeKit with my sources/styles.scss that exports the final file to css/styles.css

Using the software hooks I pick that CSS and basically runs this command
npx tailwindcss build $CK_PROJECT_ROOT/css/styles.css -o $CK_PROJECT_ROOT/css/styles.min.css

Finally, I use styles.min.css.
Normally, I just compile the sources/styles.scss directly to css/styles.min.css, using this is another step and it's getting slower and slower for each update especially when I'm developing the design of the websites. PrePros is a similar software and I believe is also have the same hooks functionality, my question is to know if there's a better way (not changing for webpack or gulp or other).

**Comment by CydGoblin** [comment]
Better support for Tailwind on Codekit may be coming https://github.com/bdkjones/CodeKit/issues/593#issuecomment-736272186

**Comment by Gutkows** [comment]
I've been trying sass pre-processors lately and the choice seems to be between PrePros vs CodeKit for me. What are your thoughts and experiences and what do you think is the better one of those two?




[MyAARPMedicare](https://www.myaarpmedicare.top/)

**Reply by Iwanttobeanonymous** [reply]
I've used both and they're very similar. I feel like Codekit has some slightly more advanced features, such as cache-busting and easier to integrate other packages (foundation, bootstrap, bourbon, etc). With that said, I'm working on windows these days, and only prepros is available. I've thought about setting up something using glup or whatever they use these days. I'm a designer, who would rather not be bothered with this aspect of managing my portfolio.

If you're on a mac, I would personally choose codekit over prepros.

**Comment by xcrap** [comment]
I've been using both for the last couple of years, Prepros is younger but has more regular updates, Codekit for me was better but I'm using PrePros now for the moment as it appears to be faster even with a small number of projects with the same settings on my 2015 Macbook. They are more or less neck to neck so my preference for Prepros now is only because it's faster, even has a strange bug that it refreshes twice while updating JS files, something annoying that does not happens with Codekit.
